ext/opcache/jit/ir/ir_x86.dasc

```diff
@@ -1064,6 +1064,7 @@ const char *ir_reg_name(int8_t reg, ir_type type)
 	_(SSE_CEIL)            \
 	_(SSE_TRUNC)           \
 	_(SSE_NEARBYINT)       \
+	_(OR_PWR2)             \
```

Comment:
Changes in ext/opcache/jit/ir will be submitted as a proper PR on IR repository.

Here I add an optimization so that

```
ir_OR(x, y)
```

is compiled to a single `bts` instruction when `y` is a large power of two. This avoids copying `y` to a register first (because `or` doesn't accept an imm64 op2), saving an instruction and reducing code size.

So this optimizes this:
``` asm
movabsq $9223372036854775808, %rcx
orq %rcx, %rax
```

Into this:
``` asm
btsq %rax, $63
```

GCC does this optimization: https://godbolt.org/z/eKd5788MY (but not Clang).

Reply:
I pushed a PR here: https://github.com/dstogov/ir/pull/111